namespaces/Representor.md

```diff
@@ -0,0 +1,107 @@
+# Representor Namespace
+
+This document extends [Refract](../refract-spec.md) Specification to describe the [Representor](https://github.com/the-hypermedia-project/charter) DOM.
+
+## Resource (Object Element)
+
+The Resource represents a Hypermedia Resource, with its available relations, transitions and attributes.
+
+### Properties
+
+- `element`: resource (string, fixed)
+- `attributes` (object)
+    - `relations` - (array[Relation]) - The relations from this resource.
+    - `transitions` - (array[Transition]) - The available transitions from this resource.
+    - `resources` - (array[Resource]) - The embedded relations (resources) from this resource.
+    - `relation` - (string) - An optional relation to this resource, when embedded in another resource.
+- `content` (object) - Any properties of the resource.
+
+## Relation (Object Element)
+
+A relation element representing a relation to another resource.
+
+### Properties
+
+- `element`: relation (string, fixed)
+- `attributes` (object, required)
+    - `relation` - (string, required) - The relation from the Resource to this resource.
+- `content` (string, required) - The URI for this relation.
+
+### Example
+
+```json
+{
+    "element": "relation",
+    "attributes": {
+      "relation": "self"
+    },
+    "content": "https://polls.apiblueprint.org/questions/1"
+}
+```
+
+## Transition (Object Element)
+
+A transition is an available progression from one state to another state.
+
+### Properties
+
+- `element`: transition (string, fixed)
+- `attributes` (object, required)
+    - `relation` - (string, required) - The relation of this transition from the current resource.
```

Comment:
I found it odd that sometimes `relation` is `string` and sometimes it means `Relation (object)`. Is this the "relation type"? But I guess it all goes down to my relation vs transition question.